src/pages/sponsors-page/SponsorsPage.tsx

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import { DisplaySponsors } from '../../components/display-sponsors/DisplaySponsors';
 import { SubpageWrapper } from '../../components/subpage-wrapper/SubpageWrapper';
+import { Helmet } from 'react-helmet';
 
 export default function SponsorsPage() {
     return(
-        <SubpageWrapper title={__('sponsorsPage.header')} bottomMargin={true}>
-            <DisplaySponsors/>
-        </SubpageWrapper>
+        <>
+            <Helmet>
```

Comment:
I think that it should be extracted into component, and reused for all pages. Maybe refactor also pages and create component with properties for page content component and object with metadata?

Reply:
fixed, i created a head wrapper so it can be used independently from layout wrappers